Client/Frontend/Settings/SettingsViewController.swift

```diff
@@ -141,7 +186,7 @@ class SettingsViewController: TableViewController {
             ]
         )
         
-        let reloadCell = { (row: Row, displayString: String) in
+        let reloadCell = { [unowned self] (row: Row, displayString: String) in
```

Comment:
Why unowned?

Reply:
This one was just moved from below (see removed `[unowned self]` at L161 because `self` was no longer used when the functionality was brought out into this block